description.md

```diff
@@ -32,6 +32,80 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interactivity is important but is limited by the larger audience: not all participants will be able to actively engage in conversations;
+however, items that don't require conversation, such as quizzes, are a good way to engage all participants, as we discuss below.
+Practical sessions, on the other hand, put participants in the position where they work on concrete items, and get involved in smaller groups discussions, with a less asymmetrical interaction.
+
+Materials are required for delivering lectures.
+In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.
+
+#### Lecture Materials
+
+We organize lecture materials according to lecture-specific activities.
+We reference them as presented in the ["Types of Content" section](#types-of-content).
+
+* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
+  They consist of:
```

Comment:
```suggestion

  They consist of:
```
Without an empty line before the indented line, there is no change in the view. Is this for stylistic reasons only?

Reply:
@Alex-deVis , I don't understand this.